<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>781</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>6</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>60</y>
        <width>521</width>
        <height>441</height>
       </rect>
      </property>
      <property name="title">
       <string>Menu</string>
      </property>
      <widget class="QPushButton" name="btnClient">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>90</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Clients</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btnParking">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>200</y>
         <width>131</width>
         <height>61</height>
        </rect>
       </property>
       <property name="text">
        <string>Parkings</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Gestion Clients</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRetour1">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>468</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QTableView" name="tvClient">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>120</y>
        <width>641</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSearchClient">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>80</y>
        <width>391</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddClient">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>460</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter Client</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbCSort">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>80</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Gestion Parkings</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRetour2">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>460</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddParking">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>450</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter Parking</string>
      </property>
     </widget>
     <widget class="QTableView" name="tvParking">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>641</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cbPSort">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>70</y>
        <width>121</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leSearchParking">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>391</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>50</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter Client</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddCRetour">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>360</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddC">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>292</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAddCLast">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>110</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAddCFirst">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>110</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>110</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAddCAdress">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>170</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAddCCIN">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>170</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>170</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAddCEmail">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>240</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>E-mail</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>250</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>E-mail</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAddCAdress_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>180</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpdateC">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>302</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>120</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAddCCIN_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>180</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>180</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CIN</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAddCEmail_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>250</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAddCFirst_2">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>120</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier Client</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRemoveC">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>370</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="leAddCLast_2">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>120</y>
        <width>161</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpdateRetour">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>440</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>50</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter Parking</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>130</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>127</y>
        <width>194</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cbClientAddP">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>188</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cbEtatAddp">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>250</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>190</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Client</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>104</x>
        <y>250</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Etat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddRetourP">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>368</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnAddP">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>300</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbClientidAddp">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>320</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <zorder>cbClientidAddp</zorder>
     <zorder>label_15</zorder>
     <zorder>label_16</zorder>
     <zorder>dateTimeEdit</zorder>
     <zorder>cbClientAddP</zorder>
     <zorder>cbEtatAddp</zorder>
     <zorder>label_17</zorder>
     <zorder>label_18</zorder>
     <zorder>btnAddRetourP</zorder>
     <zorder>btnAddP</zorder>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Etat</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbEtatUpdatep">
      <property name="geometry">
       <rect>
        <x>216</x>
        <y>270</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="cbClientUpdateP">
      <property name="geometry">
       <rect>
        <x>216</x>
        <y>208</y>
        <width>191</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpdateP">
      <property name="geometry">
       <rect>
        <x>546</x>
        <y>320</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>126</x>
        <y>150</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QDateTimeEdit" name="dteUpdateP">
      <property name="geometry">
       <rect>
        <x>216</x>
        <y>147</y>
        <width>194</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>86</x>
        <y>70</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifier Parking</string>
      </property>
     </widget>
     <widget class="QComboBox" name="cbClientidUpdatep">
      <property name="geometry">
       <rect>
        <x>576</x>
        <y>340</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRemoveP">
      <property name="geometry">
       <rect>
        <x>546</x>
        <y>388</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>126</x>
        <y>210</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Client</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpdateRetourP">
      <property name="geometry">
       <rect>
        <x>547</x>
        <y>453</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Retour</string>
      </property>
     </widget>
     <zorder>cbClientidUpdatep</zorder>
     <zorder>label_22</zorder>
     <zorder>cbEtatUpdatep</zorder>
     <zorder>cbClientUpdateP</zorder>
     <zorder>btnUpdateP</zorder>
     <zorder>label_19</zorder>
     <zorder>dteUpdateP</zorder>
     <zorder>label_20</zorder>
     <zorder>btnRemoveP</zorder>
     <zorder>label_21</zorder>
     <zorder>btnUpdateRetourP</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
